documents/src/pages/build-app/framework-integration/react.md

```diff
@@ -11,214 +11,186 @@ layout: default
 
 # React Guide
 
-@>This guideline uses create-react-app, React v17.0.2
+@>This guideline uses create-react-app, React v18.2.0
 
-## Initialise your project
+## Using web components in React
 
-To create a new app, you may choose one of the following methods:
+Current version of React does not fully support Web Components. You have to pass the properties and events in a special ways,
+the best solution is to write a React Component that behaves as a wrapper for your Web Component.
 
-### npx
+@>Experimental version of react is fully support Web Components you can try this yourself with a [live demo](https://codesandbox.io/s/tabbar-router-experimental-dq0npp?file=/src/App.js). To follow the status of it check out [custom-elements-everywhere.com](https://custom-elements-everywhere.com/#react).
 
-```sh
-npx create-react-app my-app
-```
+To overcome these shortcomings in our example, we will show how we can create thin React wrapper components around our Web Components. 
 
-It will create a directory called `my-app` inside the current folder. Inside that directory, it will generate the initial project structure and install the required dependencies.
+### Create a wrapper component
 
-Serve your app to check if the project is created correctly.
+To create React application we will use [Create React App](https://create-react-app.dev/) CLI tool. Run the following command.
 
 ```sh
-cd my-app
-npm start
+npx create-react-app my-app
 ```
 
-## Install EF elements and themes
+It will create a directory called `my-app` inside the current folder. Inside that directory, it will generate the initial project structure and install the required dependencies.
```

Comment:
```suggestion
At current directory, it will create a new folder called `my-app`. Inside the folder, it will be provided with the initial React project structure and install required dependencies.
```